Submodules/CEF

```diff
@@ -1 +1 @@
-Subproject commit a74fdafc2f42f4349d777e1a1fb20f63e848da23
+Subproject commit 0193bbffe0a032649014cf032bcea08ec58eed7f
```

Comment:
It looks like the submodule got updated somehow here, a74fdafc2f42f4349d777e1a1fb20f63e848da23 is the correct hash.